base/doc/ltnews33.tex

```diff
@@ -355,6 +355,41 @@ \subsection{Execute \cs{par} at the end of \cs{marginpar} arguments}
 
 
 
+\subsection{Providing the raw option list of packages or documentclass to key/value handlers}
+
+\LaTeXe{} has always normalized space in option lists so\\
+\verb|\documentclass[ a4paper , 12pt ]{article}|\\
+processed the intended options \texttt{a4paper} and \texttt{12pt}.
+
+Unfortunately the mechanism used was designed for the simple option
+names of the standard option processing.  Many classes and packages 
+now use extended \emph{keyval} processing, however this white space
+normalisation makes this difficult:
+\verb|[bb=1 2 3 4]|
+which might be expected to pass a bounding box of four numbers is
+normalised to \texttt{bb=1234} and
+\verb|[bb={1 2 3 4}]|\\
+which might be expected to quote the spaces results in low level \TeX{}
+parsing errors.
+
+
+For compatibility reasons, the standard option processing has not been
+changed however the original un-normalised class and option lists
+are now saved. they are not used in the standard processing, however
+extended package option systems may use these ``raw'' option list
+macros if they are defined.
+
+The one change affecting the standard processing is that the low level
+error mentioned above is now avoided as values (any tokens to the
+right of an =  sign) are removed from consideration from the ``unused
+option list''  In this release \texttt{clip=true}` and
+`\texttt{clip=false}` both contribute \texttt{clip} to the list of
```

Comment:
A missing period just after ` ``unused option list'' `, and a missing `` ` `` just before `` \texttt{clip=true}` ``.